src/Microsoft.CodeAnalysis.Analyzers/readme.md

```diff
@@ -0,0 +1,5 @@
+# Microsoft.CodeAnalysis.Analyzers
```

Comment:
NIT: it seems the paragraph usually named `About`, but have no strong opinion, up to you
```suggestion
## About
```

Reply:
Thanks, will fix with a follow-up change

Reply:
Thanks, and sorry for late review.

Reply:
@buyaa-n The format in this PR matches the existing readme for NetAnalyzers package and also readme files in the Roslyn repo for other CodeAnalysis packages. I'll stick with the current content and format to stay consistent, we can audit and update all of them in future if needed.

https://github.com/dotnet/roslyn-analyzers/blob/main/src/NetAnalyzers/readme.md